packages/modal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@freshworks/modal",
-  "version": "0.6.6",
+  "version": "0.6.7",
```

Comment:
We need not specify the versions explicitly anywhere. Release will be taken care during deployment